driver/compile.go

```diff
@@ -544,6 +544,9 @@ func copyProcs(ps []ast.Proc) []ast.Proc {
 }
 
 func buildSplitFlowgraph(branch, tail []ast.Proc, mergeField field.Static, reverse bool, N int) *ast.SequentialProc {
+	if len(branch) == 0 {
+		branch = []ast.Proc{passProc}
+	}
```

Comment:
This doesn't seem right. Wrapping multiple pass procs in a parallel proc can only impair performance.

Reply:
addressed in 6e52d7ec370fe0f40b7f554032f357f4f2930b4f . That uncovered a bug in the first & last reducers, which is addressed in 92ede89508fe82ef9b03a66e22fece961b40f4eb .